test/test_ldap.rb

```diff
@@ -91,4 +91,10 @@ def test_encryption
 
     assert_equal enc[:method], :start_tls
   end
+
+  def test_initialize
```

Comment:
Good test, but could we rename this to something more descriptive? Perhaps `test_initialize_requires_hash_encryption`?